1.max_projection/README.md

```diff
@@ -12,3 +12,18 @@ To process each plate and extract max-projected TIFF files, we must run each Cel
 3. Once in the GUI, drag the pipeline and respective slide `nd2` file into the GUI.
 4. Before running, go to the `Metadata` module and press the `Extract metadata`. This is required for CellProfiler to extract metadata from this file type.
 5. Press `Run analysis` and repeat the steps for each slide/plate.
+
+## TIFF output
+
+In this module, we extract the maximum projected images as TIFF files per site. 
+The naming convention of the files are as shown below:
+
+**Example:**
+
+`slide2_B4_M55_CH2_Z09.tiff`
+
+- ***slide1*** = slide/plate
```

Comment:
```suggestion
- ***slide2*** = slide/plate
```

Reply:
Nice catch! That was a typo on my part, thanks!